nrf_security/configs/nrf-config.h.template

```diff
@@ -1111,7 +1111,7 @@
  *      MBEDTLS_TLS_ECDH_RSA_WITH_CAMELLIA_128_GCM_SHA256
  *      MBEDTLS_TLS_ECDH_RSA_WITH_CAMELLIA_256_GCM_SHA384
  */
-#cmakedefine MBEDTLS_KEY_EXCHANGE_ECDH_RSA_ENABLED
+//#cmakedefine MBEDTLS_KEY_EXCHANGE_ECDH_RSA_ENABLED
```

Comment:
This is wrong way to fix NCSDK-7513.

There are other users of nrf security, than openthread.